ethicml/utility/lookup.py

```diff
@@ -2,24 +2,37 @@
 
 from typing import Callable, Dict
 
-from ethicml.data import Dataset, adult, compas, credit, german, nonbinary_toy, sqf, toy
+from ethicml.data import (
```

Comment:
actually, I don't completely love this circular import. `utility` should depend on no other module. `data` should only depend on `utility`. all the other modules may depend on both `data` and `utility`.